examples/.eslintrc.js

```diff
@@ -0,0 +1,58 @@
+module.exports = {
+    'extends': [
+        'eslint-config-airbnb-base',
+        'eslint-config-airbnb-base/rules/strict',
+    ],
+    parserOptions: {
+        ecmaVersion: 5,
+        sourceType: 'script',
+        ecmaFeatures: {
+            impliedStrict: true
+        }
+    },
+    env: {
+        browser: true,
+        es6: false,
+        amd: true,
+        commonjs: true
+    },
+    rules: {
+        'no-plusplus': 'off',
+        // this option sets a specific tab width for your code
+        // http://eslint.org/docs/rules/indent
+        indent: ['error', 4, {
+            SwitchCase: 1,
+            VariableDeclarator: 1,
+            outerIIFEBody: 1,
+            // MemberExpression: null,
+            // CallExpression: {
+            // parameters: null,
+            // },
+            FunctionDeclaration: {
+                parameters: 1,
+                body: 1
+            },
+            FunctionExpression: {
+                parameters: 1,
+                body: 1
+            }
+        }],
+        'one-var': ['error', 'never'],
+        'valid-jsdoc': ['error', {
+            requireReturn: false,
+            requireParamDescription: false,
+            requireReturnDescription: false,
+        }],
+
+        // deactivated rules for es5
+        'no-var': 'off',
+        'prefer-arrow-callback': 'off',
+        'object-shorthand': 'off',
+        'no-param-reassign': ['error', { 'props': false }],
+        'no-mixed-operators': ['error', { allowSamePrecedence: true }],
+
+        // TODO reactivate all the following rules
+        'no-underscore-dangle': 'off',
+
```

Comment:
>  those overrides are useless

They are not. Try to remove `prefer-arrow-callback` above.

> the removed lines here do not reactivate the corresponding rules.

They do. Try adding a very long line in a js file in examples.

The merging capability is apparently not as clever as you think it is (or I completely missed what you're saying). According to my test, the `rules` property of this file completely shadowed the `rules` property of the main `.eslintrc`.

> We should probably go for simplicity here and simply override the parserOptions and env.

Not enough, according to my tests. Eslint still complains about classic `function` callback until I disabled the `prefer-arrow-callback` above.